src/com/epam/javacourse/homework2/HomeWork.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.javacourse.homework2;
+
+import java.util.Arrays;
+
+public class HomeWork {
+
+    public static void main(String[] args) {
+        // Fibonacci
+        Fibonacci fibonacci = new Fibonacci();
+        fibonacci.getFibonacciRow(10);
+
+        //Palindrome
+        Palindrome palindrome = new Palindrome();
+        System.out.println("\nIs 121 Palindrome: " + palindrome.isPalindrome(121));
+
+        //Sqrt
+        Sqrt sqrt = new Sqrt();
+        final int mySqrt = sqrt.mySqrt(4);
```

Comment:
The variable initialization can be omitted here like on line 14.
Please, consider other initialization lines in this file too.

Reply:
Discussed.